packages/retail-ui-extensions/src/components/CameraScanner/CameraScanner.ts

```diff
@@ -1,6 +1,22 @@
 import {createRemoteComponent} from '@remote-ui/core';
 
-export interface CameraScannerProps {}
+export type BannerType =
+  | 'Success'
+  | 'Warning'
+  | 'Error'
+  | 'Info'
+  | 'Critical'
+  | 'Default';
+
+export interface Information {
+  message: string;
+  bannerType?: BannerType;
+}
+
+export interface CameraScannerProps {
+  barcodeStatus?: Information;
+  showBanner?: boolean;
+}
```

Comment:
I took a look at POS again, there's some other props like this used on the scnaner.
```
   onBannerDismiss?: () => void; 
   overlayClickableText?: string; 
   hideOverlayClickableText?: boolean; 
```
   
Is `overlayClickableText` the "Dismiss" action? Does it make sense to expose some/all of these while we're at it?